exercises/luhn/src/example.erl

```diff
@@ -34,14 +48,26 @@ checksum([H | ReversedNumber], even, Total) when H >= $5 ->
 
 
 valid(Number) ->
-  checksum(Number) rem 10 == 0.
-
-
-
-create(Number) ->
-  lists:flatten([Number, ($: - (checksum(Number ++ [$0]) rem 10))]).
+  case is_numeric(Number) of
+    true ->
+      case is_min_length(Number) of
+      	true ->
+	      case lists:filter(
```

Comment:
Can you extract this into a single function?

```erl
filter_input(Number) ->
  lists:filter(...).
```

I think that would make the `case` much more readable.